packages/tools/storybook/stories/components/select/select.web-components.stories.ts

```diff
@@ -58,7 +58,7 @@ export default {
 
 /* Default */
 const TemplateDefault = (args: any) => html`
-  <osds-select ...=${getTagAttributes(args)}>
+  <osds-select style="width:20%" ...=${getTagAttributes(args)}>
```

Comment:
avoid those on stories
- it should display default rendering
- you cannot test flex attr anymore